src/components/Events.vue

```diff
@@ -19,9 +19,10 @@
 				/>
 			</div>
 		</div>
-		<div v-else class="no-event-container">
+		<div v-else-if="isEmpty" class="no-event-container">
 			<p>No sessions in the next 5 days 💤</p>
 		</div>
+		<div v-if="!isEmpty && !hasEvents" class="loading"></div>
```

Comment:
add computed property `shouldLoadingEvents`

```suggestion
<div v-if="shouldLoadEvents" class="loading"></div>
```

```js